app/Main.hs

```diff
@@ -518,6 +518,15 @@ addFromBower name = do
     stripBowerNamePrefix s = fromMaybe s $ T.stripPrefix "purescript-" s
     mkPackageName' = Bifunctor.first show . mkPackageName . stripBowerNamePrefix
 
+formatPackageFile :: IO ()
+formatPackageFile = do
+    pkg <- readPackageFile
+    db <- readPackageSet pkg
+    writePackageSet' db
+  where
+    writePackageSet' =
+      writeTextFile "packages.json" . packageSetToJSON
```

Comment:
If we have a `readPackageFile` should we maybe also have a `writePackageFile`? Then you could define "packages.json" as a constant and use it in both definitions.

Reply:
we have a `writePackageFile`, but it writes to the psc-package.json, and then we have `readPackageSet` but it writes to packages.json within the package set you're using currently

Reply:
I guess in reality I should really be reading the packages.json file too though, yeah